app/templates/_brief_attributes.html

```diff
@@ -1,7 +1,9 @@
 {% import "toolkit/summary-table.html" as summary %}
 {% call(item) summary.list_table(
   [
-    {"label": "Published", "value": brief.publishedAt|dateformat}
+    {"label": "Published", "value": brief.publishedAt|dateformat},
+    {"label": "Deadline for asking questions", "value": brief.clarificationQuestionsPublishedBy|dateformat},
```

Comment:
This is the wrong date - the deadline should be `clarificationQuestionsClosedAt`